fonts/Makefile.am

```diff
@@ -39,10 +39,10 @@ gresym: gresym.ttf
 
 greextra: greextra.ttf
 
-clean:
-	@$(TOP_LEVEL_MAKE) $(RM) -f $(TTFFILES)
+clean-local:
+	$(RM) -f $(TTFFILES)
 
-install:
+install-data-local:
```

Comment:
`make install` calls `install-data-local`, so no behaviour change here, it's just the clean way to do what we did before